docs/README.md

```diff
@@ -48,13 +41,5 @@ In order to use an account for ACH transactions, it will need to be validated. T
 
 See more information on [how account validation strategies work](./account-validation.md).
 
-## Getting Help
```

Comment:
I think we should keep this as people will load the GitHub Pages and want to join the community or see other pages for the project.

Reply:
I agree. I will update template then as well :)